src/runtime_src/core/common/query_requests.h

```diff
@@ -3347,8 +3347,15 @@ struct xgq_scaling_temp_override : request
 
 struct performance_mode : request
```

Comment:
A brief comment for this struct would be good, but at least this one is pretty obvious. 
Something like
```suggestion
// Get and set power mode of device
struct performance_mode : request
```
I know vast majority of the structs are not commented, which in hindsight is bad.